database/seeders/ShieldSeeder.php

```
@@ -32,55 +57,33 @@ public function run(): void
             static::seedUserTenantPivot($userTenantPivot);
         }
 
         $this->command->info('Shield Seeding Completed.');
     }
 
-    protected static function seedTenants(string $tenants): void
-    {
-        if (blank($tenantData = json_decode($tenants, true))) {
-            return;
-        }
+    protected static function seedTenants(array $tenants): void {}
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$tenants'. (undefined)

(UnusedFormalParameter)